tasks/galera.yaml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Check if grastate.dat exists
+  stat:
+    path: /var/lib/mysql/grastate.dat
+  register: grastate
```

Comment:
Please use `mariadb_` prefix in `mariadb` role as registered variables can affect other roles too.

Reply:
ok

Reply:
Fixed in 528ad057e87486641185444b66bb51bcc34794f3